### ISSUE TITLE
ModuleRouterAdapter is throwing an error when initialized with default params

### ISSUE THREAD

**Issue by salah3x** [oracle/oraclejet#58]
I'm using JET 8.0.0
When I create a **ModuleRouterAdapter** and pass it a **CoreRouter** instance
`64| this.module = new ModuleRouterAdapter(router);`
I get the following exception
```
ojmodulerouter-adapter.js:201 Uncaught (in promise) TypeError: Cannot read property 'animationCallback' of undefined
    at new ModuleRouterAdapter (ojmodulerouter-adapter.js:201)
    at new RootViewModel (appController.ts:64)
    at Root.init (root.ts:16)
    at new Root (root.ts:11)
    at main.js:62
```
The error disapear if I pass an empty `options` param
`this.module = new ModuleRouterAdapter(router, {});`

The docs states that this param is optional.

**Comment by peppertech** [comment]
Hello, 

Yes, this is a known issue and should be fixed in 8.1.0.  

You have found the workaround correctly.

**Comment by salah3x** [comment]
Hi,
I just updated the app to 8.1.0 and I can confirm that there is still the same issue.

**Comment by peppertech** [comment]
This is fixed in the v8.2.0 release which came out recently.  Sorry for the confusion in the earlier post about it being in v8.1.0.
